.jenkins/actions/run_standalone.sh

```diff
@@ -78,6 +78,12 @@ if [ "${SAVE_CACHE}" == "true" ] ; then
     TIMESTEPS=2
 fi
 
+# GTC backend name fix: passed as gtc_gt_* but their real name are gtc:gt:*
+if [[ $backend = gtc_gt_* ]] ; then
+    # sed explained: replace _ with :, two times
+    backend=`echo $backend | sed 's/_/:/;s/_/:/'`
+fi
```

Comment:
we don't want gtc:gt:cpu_ifirst, but are ok with gtc:cuda?

Reply:
Added.